### PR TITLE
feat: ui revamp

### PR DESCRIPTION
**Pull request by ahmadabdolsaheb**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

This is the fist attempt to revamp the UI

Hid the dark/light switch for now for simplicity. 
Once we update the theme selector on learn we could grab the user preferences from localstorage and show it here.

### CODE REVIEW

**Comment by ShaunSHamilton** [comment]
> we could grab the user preferences from localstorage and show it here.

I wanted to do this, but we removed it: https://github.com/freeCodeCamp/freeCodeCamp/pull/43829

**Comment by ShaunSHamilton** [comment]
image

Anyone notice anything wrong....

File: src/components/logos/near-logo.tsx [hunk 1]
```diff
@@ -1,23 +1,21 @@
 import React, { SVGProps } from 'react';
+import Spacer from '../helpers/spacer';
 
-function NearLogo(props: Partial<SVGProps<SVGSVGElement>>) {
+function NearLogo({ jumbotron }: { jumbotron: boolean }) {
+  const logoText = jumbotron ? <h1>Near</h1> : <h2>Near</h2>;
```
Comment (ShaunSHamilton) [inline src/components/logos/near-logo.tsx:5]: Do we want to keep this capitalised?

```suggestion
  const logoText = jumbotron ? <h1>NEAR</h1> : <h2>NEAR</h2>;
```

File: src/components/logos/solana-logo.tsx [hunk 2]
```diff
@@ -2,7 +2,8 @@ import React, { SVGProps } from 'react';
 
 import './logos.css';
 
-function SolanaLogo(props: Partial<SVGProps<SVGSVGElement>>) {
+function SolanaLogo({ jumbotron }: { jumbotron: boolean }) {
+  const logoText = jumbotron ? <h1>Solana</h1> : <h2>Sonala</h2>;
```
Comment (ShaunSHamilton) [inline src/components/logos/solana-logo.tsx:6]:
```suggestion
  const logoText = jumbotron ? <h1>Solana</h1> : <h2>Solana</h2>;
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]

**Comment by ahmadabdolsaheb** [comment]
> image
> 
> Anyone notice anything wrong....
no :)

**Pull request merged**